src/web/pages/reports/__tests__/auditfilterdialog.jsx

```diff
@@ -0,0 +1,69 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import {describe, test, expect, testing} from '@gsa/testing';
+import React from 'react';
+
+import Capabilities from 'gmp/capabilities/capabilities';
+import {setLocale} from 'gmp/locale/lang';
+
+import Filter from 'gmp/models/filter';
+
+import {rendererWith} from 'web/utils/testing';
+
+import AuditReportFilter from 'web/pages/reports/auditfilterdialog';
+
+setLocale('en');
```

Comment:
setting the locale shouldn't be necessary anymore. It should default to 'en' nowadays. Therefore you should be able to remove this line from every test.
```suggestion
```

Reply:
Removed.